sql/moz-fx-data-shared-prod/subscription_platform_derived/logical_subscriptions_history_v1/query.sql

```diff
@@ -112,6 +118,9 @@ subscription_attributions AS (
     ON subscription_starts.mozilla_account_id_sha256 = customer_attribution_impressions.mozilla_account_id_sha256
     AND service.id IN UNNEST(customer_attribution_impressions.service_ids)
     AND subscription_starts.started_at >= customer_attribution_impressions.impression_at
+  WHERE
+    -- The SubPlat attribution impression events we have access to are only for the Stripe subscription funnel.
+    subscription_starts.provider = 'Stripe'
```

Comment:
This aligns with Santiago Andrigo's request in [DS-3815](https://mozilla-hub.atlassian.net/browse/DS-3815).